src/V3Timing.cpp

```diff
@@ -153,13 +165,10 @@ class TimingSuspendableVisitor final : public VNVisitor {
             if (!cextp->classp()->user1SetOnce()) cextp->classp()->repairCache();
             if (auto* const overriddenp
                 = VN_CAST(cextp->classp()->findMember(nodep->name()), CFunc)) {
-                if (overriddenp->user2()) {  // If it's suspendable
-                    nodep->user2(true);  // Then we are also suspendable
-                    // As both are suspendable already, there is no need to add it as our
-                    // dependency or self to its dependencies
-                } else {
-                    // Make a dependency cycle, as being suspendable should propagate both up
-                    // and down the inheritance tree
+                if (nodep->user2() < overriddenp->user2()) {
```

Comment:
I think this could use a comment. Or, as there's a few of these, a separate function(s) with a comment. Using less/greater/etc. in these is non-obvious to me. I know this is described in the `user2` comment, but it would be nicer if we didn't have to jump over there to read the code.